.travis.yml

```diff
@@ -1,4 +1,5 @@
 sudo: false
+dist: precise
```

Comment:
I tried to use trusty containers, but I ran into this issue with the mysql db https://github.com/travis-ci/travis-ci/issues/6842